### PR TITLE
readme: add more information on collaborating

### PR DESCRIPTION
**Pull request by buschtoens**
I've been creating some new labels and cataloging all issues, so that it's easier for potential contributors to get started.

I also made it clear, that we're *reeeeally* interested in new feature ideas and possible use cases for ELT.

[Rendered](https://github.com/buschtoens/ember-light-table/blob/009cb1b06818af188f9e652afc48d2d7c8d0c8a6/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,13 +30,75 @@ A lightweight contextual component based table addon that follows Ember's action
...
-- ### [Live Demo](http://offirgolan.github.io/ember-light-table)
+- :rocket: [Live Demo][demo]
+- :books: [API Documentation][docs]
+- :pencil: [Changelog](CHANGELOG.md)
 
-- ### [Documentation](http://offirgolan.github.io/ember-light-table/docs)
+## :sos: Looking for Help?
 
-- ### [Changelog](CHANGELOG.md)
+- :warning: **Bug reports**: If your bug hasn't been reported yet, please [**open an issue**][new-issue]. Try to pick a short but descriptive title. Make sure you're using the latest version of *ember-light-table*. In the issue body, try to provide exact steps for reproduction, ideally with example code. If you can't, please include any and all error messages, as many details as possible and exact information on which Ember.js / ember-cli version and browser / OS you're using.
```
Comment (alexander-alvarez) [inline README.md]: should we setup a base twiddle that people can fork?

Comment (alexander-alvarez) [inline README.md]: and then mention it here

Comment (buschtoens) [inline README.md]: Awesome idea! #442

**Comment by offirgolan** [comment]
Not gonna lie but this is beautiful. Awesome work @buschtoens!

**Pull request merged**